tests/Services/SearchServiceTest.php

```diff
@@ -1121,17 +1132,7 @@ public function redirectToProductPageOnDoSearchProvider(): array
     public function testFilterInvalidProductOnlyPriorToCertainVersion($version, bool $shouldFilter)
     {
         $searchServiceMock = $this->getSearchServiceMock(['getPluginRepository']);
-        $pluginRepositoryMock = $this->getMockBuilder(PluginRepositoryContract::class)
-            ->disableOriginalConstructor()
-            ->setMethods([])
-            ->getMock();
-        $pluginMock = $this->getMockBuilder(\Plenty\Modules\Plugin\Models\Plugin::class)
-            ->disableOriginalConstructor()
-            ->setMethods([])
-            ->getMock();
-        $searchServiceMock->method('getPluginRepository')->willReturn($pluginRepositoryMock);
-        $pluginRepositoryMock->method('getPluginByName')->willReturn($pluginMock);
-        $pluginMock->versionProductive = $version;
+        $this->pluginInfoService->method('getPluginVersion')->willReturn($version);
 
         $this->assertEquals($shouldFilter, $searchServiceMock->shouldFilterInvalidProducts());
     }
```

Comment:
Seems like I can not add any comments to files, which have been added but do not have any contents. It seems like you have commited an empty file called `versionProductive`. Do we need this or was this commited by accident?